src/Components/SearchForm/SearchForm.js

```diff
@@ -1,54 +1,58 @@
-import React from 'react';
+import React, {useState, useContext, useEffect} from 'react';
+import {SearchContext} from '../MovieListPage/MovieListPage';
 import styles from './SearchForm.module.scss';
-
 const title = 'FIND YOUR MOViE';
 const inputPlaceholder = 'What do you want to watch';
 
-class SearchForm extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            searchQuery: props.searchQuery
-        };
-    }
+const SearchForm = () => {
+    const [searchQuery, setSearchQuery] = useState("");
+    const SearchContextValues = useContext(SearchContext);
```

Comment:
I'd not recommend naming it as a capitalized variable, as it may confuse in the future (context value is just a value, not class, component, etc).